### PR TITLE
Update the accessibility statement with latest changes

### PR DESCRIPTION
**Pull request by colinbm**
### What
Update the accessibility statement with latest changes

### Why
Following the DAC report and associated fixes, updates are required.

Link to Trello card (if applicable): https://trello.com/c/pXrooZzP

### CODE REVIEW

File: source/_statement.html.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Accessibility statement for GovWifi
-This service is run by the Government Digital Service (GDS). We want as many people as possible to be able to use it. For example, that means you should be able to:
+This service is run by the Cabinet Office CDIO. We want as many people as possible to be able to use it. For example, that means you should be able to:
```
Comment (bethwalton) [inline source/_statement.html.md:2]: Remove CDIO, it doesn't work grammatically and just saying Cabinet Office is fine.

File: source/_statement.html.md [hunk 2]
```diff
@@ -11,6 +11,13 @@ We’ve also made the service text as simple as possible to understand.
 
 [AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability.
 
+## How accessible this website is
+We know some parts of this website are not fully accessible:
+
+* the ‘upload a file’ button may cause issues for some users
+* the way that screen readers announce the different regions of our pages can be confusing
+* one page links to a PDF document that is not fully accessible
```
Comment (bethwalton) [inline source/_statement.html.md:19]: Can we be more specific and say "our 'Memorandum of Understanding' is a PDF document that is not fully accessible" ? We don't just 'link' to a PDF, we own the PDF!

File: source/_statement.html.md [hunk 3]
```diff
@@ -23,36 +30,48 @@ We’re always looking to improve the accessibility of this service. If you find
 The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, contact the [Equality Advisory and Support Service (EASS)](https://www.equalityadvisoryservice.com/).
 
 ## Technical information about this service’s accessibility
-We're committed to making this service accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+The CDIO is committed to making this service accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
```
Comment (bethwalton) [inline source/_statement.html.md:33]: Why not "We're"? 

If we really don't want to say that, then I'd recommend we go with Cabinet Office.

File: source/_statement.html.md [hunk 4]
```diff
@@ -23,36 +30,48 @@ We’re always looking to improve the accessibility of this service. If you find
...
 
-Some content on our [status page](https://status.wifi.service.gov.uk/) is not compliant with the Web Content Accessibility Guidelines version 2.1 AA standard. The non-compliances are: 
-
-* an incorrect heading hierarchy, which fails WCAG 2.1A 1.3.1 success criterion (Info and Relationships)
-
-* missing level 1 headings, so users may not be able to accurately determine the structure of content on the page
+### PDFs and other documents
 
-* a missing main heading and an incorrect heading hierarchy on the status home page, which fails WCAG 2.1A 1.3.1 success criterion (Info and Relationships)
+One page links to a PDF document that is not fully accessible. This is not a fail under any WCAG 2.1 success criterion.
```
Comment (bethwalton) [inline source/_statement.html.md:41]: As suggested earlier, we don't 'link' to a PDF, we own and upload the PDF to the service. 

So I'd recommend "Our 'Memorandum of Understanding' is a PDF document..." etc

File: source/_statement.html.md [hunk 5]
```diff
@@ -23,36 +30,48 @@ We’re always looking to improve the accessibility of this service. If you find
...
 This service is partially compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard, because of the non-compliances listed below.
 
 ## Non accessible content
 
-Some content on our [status page](https://status.wifi.service.gov.uk/) is not compliant with the Web Content Accessibility Guidelines version 2.1 AA standard. The non-compliances are: 
-
-* an incorrect heading hierarchy, which fails WCAG 2.1A 1.3.1 success criterion (Info and Relationships)
-
-* missing level 1 headings, so users may not be able to accurately determine the structure of content on the page
+### PDFs and other documents
```
Comment (bethwalton) [inline source/_statement.html.md:39]: Are there any other documents? Just PDFs as a heading if that's all that's in this section. (I may just not be looking at this properly though)

File: source/_statement.html.md [hunk 6]
```diff
@@ -23,36 +30,48 @@ We’re always looking to improve the accessibility of this service. If you find
...
+*   our status page subscribe to updates forms, when submitted with incorrect data, do not inform screen reader users of the error message when it becomes available – this fails [success criterion 4.1.3: status messages](https://www.w3.org/TR/WCAG21/#status-messages)
 
-* the 'subscribe to updates' forms contain some text that does not meet the minimum colour contrast requirements, which fails WCAG 2.1AA 1.4.3 success criterion (Contrast (Minimum))
+We plan to have these issues addressed by the current supplier, or move to an alternative supplier in 2022.
 
-* the 'subscribe to updates' form close link ('x') is not descriptive enough for some users to work out its function or purpose, which fails WCAG 2.1A 2.4.4 success criterion (Link Purpose (In Context))
+## How we tested this service
 
-* the 'subscribe to updates' forms do not inform screen reader users of the error message that appears after incorrect data has been submitted, which fails WCAG 2.1AA 4.1.3 success criterion (Status Messages)
+The website was tested in May 2021 by DAC (digital accessibility centre) using a combination of automated and manual tests, including:
```
Comment (bethwalton) [inline source/_statement.html.md:61]: Can we change to "by the Digital Accessibility Centre (DAC)" please? It's the GDS style for writing organisations and acronyms.

File: source/_statement.html.md [hunk 7]
```diff
@@ -23,36 +30,48 @@ We’re always looking to improve the accessibility of this service. If you find
...
 
-We plan to have these issues addressed by the current supplier, or move to an alternative supplier in 2021.
+* JAWS 18 screen reader with Chrome
+* NVDA screen reader with Firefox
+* Dragon naturally speaking 15 with IE 11
+* ZoomText 11 (Fusion) with Chrome
+* Ensuring the service can be used without a mouse – Chrome and/or Edge (Chromium)
```
Comment (bethwalton) [inline source/_statement.html.md:67]: Bullet points should start with a lower case letters as they continue the sentence from the lead in line (unless it's a brand name like JAWS obviously) - can we update?

File: source/_statement.html.md [hunk 8]
```diff
@@ -23,36 +30,48 @@ We’re always looking to improve the accessibility of this service. If you find
...
-* the 'subscribe to updates' form close link ('x') is not descriptive enough for some users to work out its function or purpose, which fails WCAG 2.1A 2.4.4 success criterion (Link Purpose (In Context))
+## How we tested this service
 
-* the 'subscribe to updates' forms do not inform screen reader users of the error message that appears after incorrect data has been submitted, which fails WCAG 2.1AA 4.1.3 success criterion (Status Messages)
+The website was tested in May 2021 by DAC (digital accessibility centre) using a combination of automated and manual tests, including:
 
-We plan to have these issues addressed by the current supplier, or move to an alternative supplier in 2021.
+* JAWS 18 screen reader with Chrome
+* NVDA screen reader with Firefox
+* Dragon naturally speaking 15 with IE 11
```
Comment (bethwalton) [inline source/_statement.html.md:65]: I think this is a brand name, so it should be "Dragon NaturallySpeaking 15" 

And we should spell out acronyms, so "with Internet Explorer 11"

File: source/_statement.html.md (hunk 6 above)
Comment (bethwalton) [inline source/_statement.html.md:61]: Can we take another look at this section? The lead in line makes you expect a list of different types of tests, but then includes things like "Colour Blind, Dyslexic, Learning Disabilities, Deaf, Aspergers, and Anxiety/Panic disorder". This doesn't follow on grammatically from the lead in line, and also is not a type of test. Happy to work with whoever wrote this to make this clearer!

**Review by bethwalton — CHANGES_REQUESTED** [review]
Hi, I've added some comments throughout. Most are small points on style, formatting and clarity, but there's one section that I think needs a bit more work. It's the section about the types of work that DAC carried out. I'm happy to help with making this clearer - just give me a shout next week.

File: source/_statement.html.md (hunk 1 above)
Comment (colinbm) [reply source/_statement.html.md:2]: Gone with the latter... further discussion welcome :)

File: source/_statement.html.md (hunk 3 above)
Comment (colinbm) [reply source/_statement.html.md:33]: Left as is due to the above... again, we can revisit.

File: source/_statement.html.md (hunk 6 above)
Comment (colinbm) [reply source/_statement.html.md]: The list of disabilities should have been associated with the previous bullet, however it still didn't read great in the source material... I've included them in brackets after the word _disability_. You may have more thoughts on this!

Comment (colinbm) [reply source/_statement.html.md]: The list of disabilities should have been associated with the previous bullet, however it still didn't read great in the source material... I've included them in brackets after the word _disability_. You may have more thoughts on this!

File: source/_statement.html.md (hunk 1 above)
Comment (colinbm) [reply source/_statement.html.md]: Should the brackets be reversed?

File: source/_statement.html.md [hunk 9]
```diff
@@ -11,6 +11,13 @@ We’ve also made the service text as simple as possible to understand.
 
 [AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability.
 
+## How accessible this website is
+We know some parts of this website are not fully accessible:
+
+* the ‘upload a file’ button may cause issues for some users
```
[the remark this answers is not in the document]
Comment (rorylangford) [reply source/_statement.html.md:17]:
```suggestion
* the ‘upload a file’ button located in the sign the memorandum of understanding page may cause issues for some users
```

**Review by bethwalton — DISMISSED** [review]

**Review by bethwalton — APPROVED** [review]

**Pull request merged**